search-handlers/src/test/java/no/unit/nva/search/UserBasedResourceSearchHandlerTest.java

```diff
@@ -0,0 +1,139 @@
+package no.unit.nva.search;
```

Comment:
Tests that mocks the actual OpenSearch client verifying that the generated query has the expected paramters and filters.